roles/influxdb_exporter/vars/main.yml

```
@@ -0,0 +1,13 @@
+---
+go_arch_map:
+  i386: '386'
+  x86_64: 'amd64'
+  aarch64: 'arm64'
+  armv7l: 'armv7'
+  armv6l: 'armv6'
+
+go_arch: "{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}"
```

Please use the following instead, so that the variable name is prefixed with the role name.
```suggestion
_influxdb_exporter_go_ansible_arch: "{{ {'i386': '386',
                                'x86_64': 'amd64',
                                'aarch64': 'arm64',
                                'armv7l': 'armv7',
                                'armv6l': 'armv6'}.get(ansible_architecture, ansible_architecture) }}"
```